### ISSUE TITLE
Using tutorial over several widgets

### ISSUE THREAD

**Issue by SalahAdDin** [SimformSolutionsPvtLtd/flutter_showcaseview#113]
**Is your feature request related to a problem? Please describe.**
Our current application is splitted on different widgets following a clean architecture; we need, for example, add a notification button at the appboard to the show cases, also, we want to add a gesture widget which contains the user avatar and name to the show cases, and other similar stuff.

**Describe the solution you'd like**
If we add global keys to those widgets, it would be great if the tutorial show also those widgets in the same workflow instead to need to create a new tutorial on a different widget.

**Describe alternatives you've considered**
`tutorial_coach_mark` [does not provide this feature](https://github.com/RafaelBarbosatec/tutorial_coach_mark/issues/69).

**Additional context**

**Comment by vatsaltanna** [comment]
Hello, @SalahAdDin please confirm my understanding, So you are requesting functionality like showcasing multiple widgets in different places at the same time.

**Comment by SalahAdDin** [comment]
Yeah, we structured the main screen with different widget, some of them have components we want to showcase from the main screen, for instance.

**Comment by vatsaltanna** [comment]
Got it, Thanks for the information. I have added this as an enhancement, we will update here as soon as there will be any progress on this.

**Comment by sbis04** [comment]
Any timeline for when would this feature be implemented or whether it's in progress?

**Comment by desmeit** [comment]
i need this feature too. It is recommended by Flutter to work with multiple widgets. I think it is an important basic feature. I can't imagine anyone working with only one widget.

**Comment by vatsaltanna** [comment]
> Any timeline for when would this feature be implemented or whether it's in progress?

Hi @sbis04, We haven't started working on this yet but as it seems that to implement this functionality, it will require large scope structure change in package so we will do some feasibility check first. so I cann't commit any ETA for this. We will update here as soon as we start work on this.

**Comment by vivekprasad99** [comment]
@vatsaltanna  any update on this. any ETA

**Comment by pawanbangar** [comment]
@vivekprasad99 i have implemented this feature for multiple screens using global state, tell me if you want to do the same?
